src/pages/markets/cryptocurrencies/index.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { Hero } from '../components/sections/_hero'
+import NavTab from '../components/sections/_nav-tab'
+import Cryptocurrencies from '../components/markets/_cryptocurrencies.js'
+import Signup, { Appearances } from 'components/custom/signup'
+import Layout from 'components/layout/layout'
+import { localize, Localize, WithIntl } from 'components/localization'
+import { SEO } from 'components/containers'
+import PractiseIcon from 'images/svg/aim.svg'
+import TradeIcon from 'images/svg/trade.svg'
+import WithdrawIcon from 'images/svg/withdraw.svg'
+
+const simple_step_content_cryptocurrencies = [
+    {
+        header: <Localize translate_text="Practise" />,
+        text: (
+            <Localize translate_text="Open a demo account and practise with an unlimited amount of virtual funds." />
+        ),
+        icon: <img src={PractiseIcon} alt="aim" width="32" height="32" />,
+    },
+    {
+        header: <Localize translate_text="Trade" />,
+        text: (
+            <Localize translate_text="Open a real account, make a deposit, and start trading commodities and other markets. " />
+        ),
+        icon: <img src={TradeIcon} alt="trade" width="32" height="28" />,
+    },
+    {
+        header: <Localize translate_text="Withdraw" />,
+        text: (
+            <Localize translate_text="Conveniently withdraw your funds through any of our supported withdrawal methods." />
+        ),
+        icon: <img src={WithdrawIcon} alt="withdraw" width="32" height="32" />,
+    },
+]
+
+const CryptocurrenciesPage = () => (
+    <Layout>
+        <SEO
+            description={localize(
+                'Learn about the markets that you can trade online with Deriv, including forex, commodities, synthetic indices, and stock indices.',
+            )}
+            title={localize('Markets | Markets to trade | Deriv')}
+        />
+        <Hero />
+        <NavTab route_from={'cryptocurrencies'} route_offset={360} />
```

Comment:
```suggestion
        <NavTab route_from={'cryptocurrencies'} route_offset={480} />
```

Reply:
> 
`route_offset` is created to set x-offset for horizontal scroll (nav market tab) in mobile, as for crypto, set it to 480 should make it auto scroll to last tab..

Reply:
@kevinw-binary I think u need to change it again maybe from 480 to maybe 600 because the nav is not fully showing the tab on load. As image below. 

![image](https://user-images.githubusercontent.com/78890281/120130731-7a6e5280-c1f9-11eb-9852-5b7ea9c73e92.png)